pyproject.toml

```
@@ -40,8 +40,8 @@ packaging = ">=15.0"
 typing-extensions = "^4.5.0"
 ansys-api-acp = "^0.1.dev7"
 ansys-tools-path = "^0"
-ansys-tools-local-product-launcher = "^0.1.0"
-ansys-tools-filetransfer = "^0.1.0"
+ansys-tools-local-product-launcher = "^0.2.0"
+ansys-tools-filetransfer = "^0.2.0"
```

```suggestion
ansys-tools-local-product-launcher = ">=0.1,<0.3"
ansys-tools-filetransfer = ">=0.1,<0.3"
```
We'll need to keep allowing the released `0.1` versions. Only extending up to `0.2` to be on the safe side w.r.t introducing backwards-incompatible changes to `-filetransfer` and `-local-product-launcher`.